.zuul.yaml

```diff
@@ -215,6 +228,23 @@
 - project-template:
     name: opendev-watcher-edpm-pipeline
     openstack-check:
+      debug: true
       jobs:
-        - openstack-meta-content-provider-master
-        - watcher-operator-validation-master
+        - openstack-meta-content-provider-master:
+            # Note(chandankumar): Add override checkout at project template
+            # pipeline so that RDO zuul discovers the job and complete the job
+            # graph in order to run the job.
+            override-checkout: main
+            required-projects:
+              - github.com/openstack-k8s-operators/watcher-operator
+            # Build watcher-operator always in meta content provider
+            vars:
+              cifmw_operator_build_operators:
+                - name: mariadb-operator
+                  src: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/mariadb-operator"
+                - name: openstack-operator
+                  src: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/openstack-operator"
```

Comment:
Is this correct? mariadb-operator and openstack-operator? don't we need to add watcher-operator?

Reply:
good catch! fixed it now.